xml/admin_operating_services.xml

```diff
@@ -10,245 +10,204 @@
+mon.ses-min2                        ses-min2    running (5d)   8m ago
+mon.ses-min4                        ses-min4    running (5d)   7m ago
+osd.0                               ses-min2    running (61m)  8m ago
+osd.1                               ses-min3    running (61m)  7m ago
+osd.2                               ses-min4    running (61m)  7m ago
+rgw.myrealm.myzone.ses-min1.kwwazo  ses-min1    running (5d)   8m ago
+rgw.myrealm.myzone.ses-min2.jngabw  ses-min2    error          8m ago
 </screen>
-   <para>
-    To start/stop/restart all OSDs on the node, run:
-   </para>
+
+  <para>
+   To identify a service on a specific node, run:
+  </para>
+
+<screen>ceph orch ps <replaceable>NODE_HOST_NAME</replaceable></screen>
+
+  <para>
+   For example:
+  </para>
+
 <screen>
-&prompt.root;systemctl start ceph-osd.target
-&prompt.root;systemctl stop ceph-osd.target
-&prompt.root;systemctl restart ceph-osd.target
+&prompt.cephuser;ceph orch ps ses-min2
+NAME                                HOST      STATUS         REFRESHED
+mgr.ses-min2.oopvyh                 ses-min2  running (5d)   3m ago
+mon.ses-min2                        ses-min2  running (5d)   3m ago
+osd.0                               ses-min2  running (67m)  3m ago
 </screen>
-   <para>
-    Commands for the other targets are analogous.
-   </para>
-  </sect2>
 
-  <sect2 xml:id="ceph-operating-services-individual">
-   <title>Starting, Stopping, and Restarting Individual Services</title>
+  <para>
+   After you know the name of the service, you can start, restart, or stop it:
+  </para>
+
+<screen>ceph orch daemon <replaceable>COMMAND</replaceable> <replaceable>SERVICE_NAME</replaceable></screen>
+
+  <para>
+   For example, to restart the OSD service with ID 0, run:
+  </para>
+
+<screen>&prompt.cephuser;ceph orch daemon restart osd.0</screen>
+ </sect1>
+ <sect1 xml:id="cha-ceph-operating-service-types">
+  <title>Operating Service Types</title>
+
+  <para>
+   If you need to operate a specific type of service across the whole &ceph;
+   cluster, use the following command:
+  </para>
+
+<screen>ceph orch <replaceable>COMMAND</replaceable> <replaceable>SERVICE_TYPE</replaceable></screen>
+
+  <para>
+   For example, the following command restarts all MONs in the cluster,
+   regardless on which nodes they actually run:
+  </para>
+
+<screen>&prompt.cephuser;ceph orch restart mon</screen>
+ </sect1>
+ <sect1 xml:id="cha-ceph-operating-node">
+  <title>Operating Services on a Single Node</title>
+
+  <para>
+   By using the <command>systemctl</command> command, you can operate &ceph;
+   related &systemd; services and targets on a single node.
+  </para>
+
+  <sect2 xml:id="ceph-operating-services-finding-names">
+   <title>Identifying Services and Targets</title>
    <para>
-    You can operate individual services using the following parameterized
-    &systemd; unit files:
+    Before operating &ceph; related &systemd; services and targets, you need to
```

Comment:
The `NAME` column of `orch ps` is actually the daemon name that can be used with `systemctl`

Reply:
i've tried it now and while `ceph orch ps doc-ses-min2` returns `mon.doc-ses-min2`, `systemctl restart mon.doc-ses-min2` returns an error

Reply:
what I mean is 

`systemctl status ceph-$fsid@$NAME`

Reply:
great, thanks